trigger/rest/trigger.go

```diff
@@ -219,10 +219,19 @@ func newActionHandler(rt *RestTrigger, handler *trigger.Handler) httprouter.Hand
 				replyCode = 200
 			}
 			w.WriteHeader(replyCode)
-			if err := json.NewEncoder(w).Encode(replyData); err != nil {
-				log.Error(err)
+			switch t := replyData.(type) {
```

Comment:
@lixingwang: what are possible return types apart from string/object?

Reply:
@vijaynalawade it can be any type(Long, Double, params, object...).    just make sure write back to response writer directlly if it is a string type , others go with encoder.

Reply:
Encoder encode json string (string type) to escaped string which is not good.